csharp-advanced-topics/PointersInC#/PointersInC#/Method/Methods.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PointersInCSharp.Method;
+
+public static class Methods
+{
+    public static unsafe void SwapChars(char* p1, char* p2) => (*p2, *p1) = (*p1, *p2);
+    public static unsafe void DoubleOddValues(int* arrayPtr, int length)
+    {
+        for (int i = 0; i < length; i++)
```

Comment:
Can you move the `fixed` statement into this method instead? Currently, the calling method and 3 tests all need to fix the variable.. by moving it into this method, you can DRY up the code a bit. Thanks.